nl_server/loader.py

```diff
@@ -110,7 +110,7 @@ def _maybe_load_custom_dc_yaml():
 # (deploy/nl/).
```

Comment:
nit: ... in "custom DC" and test-like ...

Reply:
Done.